invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Breadcrumb } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+export default class Breadcrumbs extends Component {
+  constructor(props) {
+    super(props);
+    this.elements = props.elements;
+  }
```

Comment:
I find these constructors just noise, you can access directly `this.props.elements`